docs/Users/Install.rst

```diff
@@ -166,6 +166,28 @@ https://docs.python.org/3.3/install/#alternate-installation.
     http://stackoverflow.com/questions/14149422/using-pip-behind-a-proxy
     for further clarification.
 
+Error handling
+~~~~~~~~~~~~~~~~
```

Comment:
the same, this commit is already acked and I rebased on it because it's just not merged yet